code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -294,47 +303,47 @@
 			if(H.check_body_part_coverage(MOUTH))
 				to_chat(user,"<span class='warning'>The mask is in the way!</span>")
 				return
-			if(H.f_style == "Shaved")
+			/*if(H.f_style == "Shaved")
 				to_chat(user,"<span class='warning'>Already clean-shaven!</span>")
-				return
+				return*/
 
 			if(H == user) //shaving yourself
-				user.visible_message("[user] starts to shave their facial hair with [src].", \
-									 "<span class='notice'>You take a moment to shave your facial hair with [src]...</span>")
+				user.visible_message("[user] starts to style their facial hair with [src].", \
+									 "<span class='notice'>You take a moment to style your facial hair with [src]...</span>")
 				if(do_after(user, H, 5))
```

Comment:
Or put shave(H, user, location) here and handle the shaving in shave(), that way you can give them the prompt, then do the do_after after they've decided.